### PR TITLE
WIP #2058 status bar: first attempt at logic to change text on status bar

### PR DESCRIPTION
**Pull request by phillimorland**
This was my attempt at doing the story
Slightly messed up with the commits but it is working

### CODE REVIEW

File: meinberlin/apps/cms/templates/meinberlin_cms/includes/storefronts.html [hunk 1]
```diff
@@ -32,7 +32,13 @@ <h2 class="storefront__proj-text {% if item.project.name|length > 95 %} storefro
...
         </div>
-        <span class="status-bar__status"><i class="fas fa-clock"></i>{% blocktrans with time_left=item.project.time_left %} {{ time_left }} remaining {% endblocktrans %}</span>
+        <span class="status-bar__status"><i class="fas fa-clock"></i>
+          {% if (item.project.time_left < 365) %}
+          {% blocktrans with time_left=item.project.time_left %} {{ time_left }} remaining {% endblocktrans %}
+          {% else %}
+          <span>boo</span>
+          {% endif %}
+        </span>
     </div>
```
Comment (phillimorland) [inline meinberlin/apps/cms/templates/meinberlin_cms/includes/storefronts.html]: @MagdaN this was my first attempt at adding the logic to the django, i removed it in the next commit but was just wondering if I was even vaguely on the right track?

File: meinberlin/apps/plans/assets/PlansList.jsx [hunk 2]
```diff
@@ -11,6 +11,19 @@ class PlansList extends React.Component {
     return { width: item.active_phase[0] + '%' }
   }
 
+  getTimespan (item) {
+    let timeRemaining = parseInt(item.active_phase[1])
```
[the remark this answers is not in the document]
Comment (rmader) [reply meinberlin/apps/plans/assets/PlansList.jsx]: hm does the string ('23 days left') come from django? If so, should we maybe move the whole parsing to js? This looks really error prone -> what happens if there are 3 minutes left?

[the remark this answers is not in the document]
Comment (MagdaN) [reply meinberlin/apps/plans/assets/PlansList.jsx]: the python code get seconds and always returns the highest possible number + unit. It used to include also weeks and and months but at some point we said days should be the highest unit. I would leave it in python as we are using it quite often in python code.

What problem do you see with three minutes?

[the remark this answers is not in the document]
Comment (rmader) [reply meinberlin/apps/plans/assets/PlansList.jsx]: Well, if I read the code right, three minutes and three days can't get differentiated. It's assumed that the unit is 'days'

[the remark this answers is not in the document]
Comment (MagdaN) [reply meinberlin/apps/plans/assets/PlansList.jsx]: ah ok, but I guess that should not be a problem as minutes never get higher then 365

**Pull request merged**